### ISSUE TITLE
Update for jekyll 3.1.x

### ISSUE THREAD

**Issue by alias1** [bwillis/jekyll-github-sample#2]
It would be cool to update this for jekyll 3. Currently getting an error when I try to bundle

```
jekyll_github_sample was resolved to 0.1.0, which depends on
      jekyll (~> 1.0)
```

**Comment by sunieldalal** [comment]
Does it require lot of steps to make this compatible with Jekyll 3.x? I am new to ruby / jekyll but can help in any way to implement this

**Comment by bwillis** [comment]
Sorry for the long delay, I updated dependencies and released this in v0.2 🌈
